### DISCUSSION TITLE
Final Exam

### DISCUSSION THREAD

**Discussion by thaneekan** [SOEN345-WINTER2024/discussion#23] [General]
Hi, I attended the exam review but just wanted to clarify, program synthesis is not on the final exam whatsoever?

**Comment by stan6** [comment]
Yes, it is stated clearly in the review slide that program synthesis is not in the slide.

**Comment by mario-elshaer** [comment]
Hello Professor, 
I understand program synthesis is not on the final exam. Howvever, I just want to make sure which ppts cover it. 
Correct me if I am wrong but:
Lecture 11pt2
and Lecture 12  are all Program synthesis.

Is programming by example also program synthesis or not?

**Reply by stan6** [reply]
The topics under program synthesis (Lecture 12) will not be coverred.

**Comment by stan6** [comment]
We do not plan to answer any question regarding whether a specific topic will or will not be covered. 

You are encourage to review the exam review slide. Anything that is not in the exam review slide may or may not be covered.
